src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -93,15 +93,32 @@ int halide_hexagon_remote_load_library(const char *soname, int sonameLen,
 }
 
 volatile int power_ref_count = 0;
+static volatile void *power_context = NULL;
+
+static void *get_HAP_power_context() {
+    if (power_context == NULL) {
+        power_context = (void *)malloc(1);
+        log_printf("get_HAP_power_context: %p\n", power_context);
```

Comment:
I don't think we should be doing log_printf in non-error cases.

Reply:
Done.  Will push up another patch.